site-manage/site-group-manager/src/main/java/org/sakaiproject/groupmanager/service/SakaiService.java

```diff
@@ -42,6 +51,9 @@
 @Slf4j
 public class SakaiService  {
```

Comment:
super generic and vague name SakaiService, also given that this is in a tool this shouldn't be named a service as it's misleading!

maybe something like:
GroupManagerOperations
GroupManagerActions

Reply:
I don't agree with this, this is a "service that interacts with some Sakai services", SakaiService makes sense to me. I'm not going to change the name of the service and all the variables in this PR.